plugins/midas theme/themes/midas theme.json

```diff
@@ -0,0 +1,36 @@
+{

```

Comment:
Please change the file name to not include a space, e.g.
- midas_theme.json
- midasTheme.json